HousingRepairsOnlineApi/UseCases/RetrieveAvailableAppointmentsUseCase.cs

```diff
@@ -36,6 +40,8 @@ public RetrieveAvailableAppointmentsUseCase(IAppointmentsGateway appointmentsGat
             var sorEngine = sorEngineResolver.Resolve(repairType);
             var repairCode = sorEngine.MapToRepairTriageDetails(repairLocation, repairProblem, repairIssue);
 
+            var allowedAppointmentSlots = allowedAppointmentsFactory.CreateAllowedAppointments(repairType);
```

Comment:
Do we need to pass in the repair type, knowing that it's only the communal repair type that will use this functionality - or is this added to future proof the automatic appointment booking functionality?

Reply:
Yes, this is for future proofing - it allows [registering a new filter](https://github.com/Newarkandsherwood/housing-repairs-online-api/pull/42/files#diff-102bd3b66a2a275e882bc9aa0b09b739d27a714cef6b777ef2519f2d15610630R67) that will provide appointments for each specific repair type.
It means that this class and method is not aware which appointment slots are allowed for each repair type.